public/app/core/components/TraceToProfiles/TraceToProfilesSettings.tsx

```diff
@@ -0,0 +1,222 @@
+import { css } from '@emotion/css';
+import React, { useEffect, useMemo, useState } from 'react';
+import { useAsync } from 'react-use';
+
+import {
+  DataSourceJsonData,
+  DataSourceInstanceSettings,
+  DataSourcePluginOptionsEditorProps,
+  updateDatasourcePluginJsonDataOption,
+} from '@grafana/data';
+import { ConfigSection } from '@grafana/experimental';
+import { getDataSourceSrv } from '@grafana/runtime';
+import { InlineField, InlineFieldRow, Input, InlineSwitch } from '@grafana/ui';
+import { ConfigDescriptionLink } from 'app/core/components/ConfigDescriptionLink';
+import { DataSourcePicker } from 'app/features/datasources/components/picker/DataSourcePicker';
+import { ProfileTypesCascader } from 'app/plugins/datasource/grafana-pyroscope-datasource/QueryEditor/ProfileTypesCascader';
+import { PyroscopeDataSource } from 'app/plugins/datasource/grafana-pyroscope-datasource/datasource';
+import { ProfileTypeMessage } from 'app/plugins/datasource/grafana-pyroscope-datasource/types';
+
+import { IntervalInput } from '../IntervalInput/IntervalInput';
+import { TagMappingInput } from '../TraceToLogs/TagMappingInput';
+import { getTimeShiftLabel, getTimeShiftTooltip, invalidTimeShiftError } from '../TraceToLogs/TraceToLogsSettings';
+
+export interface TraceToProfilesOptions {
+  datasourceUid?: string;
+  tags?: Array<{ key: string; value?: string }>;
```

Comment:
Are we doing a version with tags? I thought we are waiting for the span API because they cannot mix anyway.